src/SAWScript/Crucible/LLVM/Override.hs

```diff
@@ -660,6 +662,45 @@ executeCond opts sc cc cs ss = do
             (ss ^. MS.csFreshPointers)
   OM (setupValueSub %= Map.union ptrs)
 
+  liftIO . mapM_ print $ cs ^. MS.csPreState . MS.csAllocs
+
+  sym <- use syminterface
+  mem <- readGlobal . Crucible.llvmMemVar $ cc ^. ccLLVMContext
+  sub <- use setupValueSub
+
+  let mutableAllocs = Map.filter (view isMut) $ cs ^. MS.csPreState . MS.csAllocs
+      allocPtrs =
+        (\(ptr, spec) ->
+           ( ptr
+           , _allocSpecBytes spec
+           , "state of memory allocated in precondition not described in postcondition"
```

Comment:
These messages should indicate which override they are from, to aid in debugging when something goes wrong. See e.g. `ppMethodSpec`:
```haskell
PP.text "Location: " <> PP.pretty (plSourceLoc (methodSpec ^. csLoc))
```